src/hotspot/share/runtime/threads.cpp

```diff
@@ -1181,10 +1182,12 @@ void Threads::metadata_handles_do(void f(Metadata*)) {
   threads_do(&handles_closure);
 }
 
-// Get count Java threads that are waiting to enter the specified monitor.
+#if INCLUDE_JVMTI
```

Comment:
Is it needed to update thread.hpp also?

Reply:
Thank you for the comment, will check it.